spec/dummy/config/environments/test.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.configure do
+  config.active_storage.service = :test
+  config.action_mailbox.ingress = :ses
+  config.action_mailbox.ses.subscribed_topics = %w(
+    arn:aws:sns:eu-west-1:012345678910:example-topic
+    arn:aws:sns:eu-west-1:012345678910:recognized-topic
+  )
+
+  config.action_dispatch.show_exceptions = :none
+  config.active_job.queue_adapter = :test
```

Comment:
I feel like we should have the queue adapter be test so mailbox can be tested in isolation from it—I hope the sqs tests aren't like silently not testing anything now

Reply:
I'll look into this.